int64 init_mode
float32 x
float32 y
float32 angle

# Constants
int64 START_RIGHT=0
int64 LEFT_HALF=1
int64 RIGHT_HALF=2
int64 POSITION=3
int64 POSE=4

---
bool success

